docs/examples/contours/canny_edge_demo.jl

```diff
@@ -0,0 +1,91 @@
+# ---
+# cover: assets/corner.png
```

Comment:
We need to generate the cover image in this script. Please see other available demo files for how it's done.

Reply:
> Is this the demo you're working on wrt to issue [JuliaImages/ImageEdgeDetection.jl#21](https://github.com/JuliaImages/ImageEdgeDetection.jl/issues/21) and [JuliaImages/ImageDraw.jl#50](https://github.com/JuliaImages/ImageDraw.jl/issues/50)?
Actually,I'm creating small examples for ideas of demos for my proposal...circle detection/measurement/processing is related to JuliaImages/ImageEdgeDetection.jl#21 and JuliaImages/ImageDraw.jl#50